src/decision/engine.js

```diff
@@ -66,7 +66,7 @@ module.exports = class Engine {
         return Boolean(nextTask)
       },
       (next) => {
-        log('got task', nextTask)
+        log('got task')
```

Comment:
There is already so much logging, that's fine. and this version was printing the whole struct which was complete overkill